JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -125,28 +136,26 @@ private fun DrawerButton(
         colors.surface
     }
 
-    val surfaceModifier = modifier +
-            LayoutPadding(start = 8.dp, top = 8.dp, end = 8.dp, bottom = 0.dp) +
-            LayoutWidth.Fill
+    val surfaceModifier = modifier
+                .padding(start = 8.dp, top = 8.dp, end = 8.dp, bottom = 0.dp)
```

Comment:
code formatting, limit padding to 4 spaces

Reply:
Thanks!